docker/standalone-ha.xml

```diff
@@ -220,7 +220,7 @@
             <log-system-exceptions value="true"/>
         </subsystem>
         <subsystem xmlns="urn:jboss:domain:io:1.1">
-            <worker name="default"/>
+            <worker name="default" io-threads="32" task-max-threads="512"/>
```

Comment:
We can remove it, but I thought this might improve situation a bit.
Default values are 4 for `io-threads` and 15 for `task-max-threads`. On WF 10 `io-threads` are calculated based on `cpuCount * 2`.

Selected numbers are non scientific ;) But the higher they are potentially more requests we can handle.